src/main/java/com/currencycloud/client/model/Account.java

```diff
@@ -234,29 +239,38 @@ public void setSettlementType(String settlementType) {
 
     @Override
     public String toString() {
-        return new JSONObject()
-                .appendField("id", id)
-                .appendField("legalEntityType", legalEntityType)
-                .appendField("accountName", accountName)
-                .appendField("brand", brand)
-                .appendField("yourReference", yourReference)
-                .appendField("status", status)
-                .appendField("street", street)
-                .appendField("city", city)
-                .appendField("stateOrProvince", stateOrProvince)
-                .appendField("country", country)
-                .appendField("postalCode", postalCode)
-                .appendField("spreadTable", spreadTable)
-                .appendField("createdAt", createdAt)
-                .appendField("updatedAt", updatedAt)
-                .appendField("identificationType", identificationType)
-                .appendField("identificationValue", identificationValue)
-                .appendField("shortReference", shortReference)
-                .appendField("apiTrading", apiTrading)
-                .appendField("onlineTrading", onlineTrading)
-                .appendField("phoneTrading", phoneTrading)
-                .appendField("processThirdPartyFunds", processThirdPartyFunds)
-                .appendField("settlementType", settlementType)
-                .toString();
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
it is more succinct to do

final ObjectMapper objectMapper = new ObjectMapper().disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS).setDateFormat(new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssX"));

Also date format string should probably be defined as a private static final data member or even as a global static property as it is defined in many classes

Reply:
Agreed. Will implement as suggested